examples/00_additive_single_bead.py

```diff
@@ -47,7 +47,7 @@
     SingleBeadInput,
 )
 
-additive = Additive()
+additive = Additive(host='127.0.0.1', port="50052", log_file="single_bead.log")
```

Comment:
Let's revert this.

Reply:
Fixed in a81154a31271562c460f2d7fddc7a3b2a6b1398c